pkg/crypto/crypto.go

```diff
@@ -0,0 +1,77 @@
+/*
+	Package crypto provides functions to encode and decode data
+	Using aes crypto
+	Random flag defines random nonce sequence or same
```

Comment:
I assume that making fields non-exported should influence this documentation as well so just get rid of random flag description since it will not be visible for package users.